scripts/create_ea_defs.sh

```diff
@@ -11,4 +11,4 @@ SSL_VERIFY="false"
 
 
 
-./create_ea_defs --grid-host=${GRID_HOST} --wapi-port=${WAPI_PORT} --wapi-username=${WAPI_USERNAME} --wapi-password=${WAPI_PASSWORD} --wapi-version=${WAPI_VERSION} --ssl-verify=${SSL_VERIFY}
+bin/create_ea_defs --grid-host=${GRID_HOST} --wapi-port=${WAPI_PORT} --wapi-username=${WAPI_USERNAME} --wapi-password=${WAPI_PASSWORD} --wapi-version=${WAPI_VERSION} --ssl-verify=${SSL_VERIFY}
```

Comment:
This seems to make the `create_ea_defs.sh` script rather inconvenient to use - you have to run the script in the parent directory of `bin` where the `create_ea_defs` executable is located.

One suggestion would be to remove the `bin` above, and move the `create_ea_defs.sh` script to the `bin` directory.